### ISSUE TITLE
Usage instructions?

### ISSUE THREAD

**Issue by ghost** [stack360/faas-ecs#1]
Hi,
First off, great work and it's amazing to see that this project exists! I'm very excited to use ECS as a backend for OpenFaaS.

Would it be possible to put a short list of step-by-step instructions in the README file so that I can try out (and hack on) this project?

Is it just a matter of replacing the `gateway/` directory in the `faas` repo with this repo's root directory?

Thanks again,
Eric

**Comment by xichengc** [comment]
Hi Eric,

Thanks for your comments. faas-ecs project is still under development and does not quite work yet. I am still working on it and will update the README once some basic functionalities are available.

Xicheng

**Comment by alexellis** [comment]
> Is it just a matter of replacing the gateway/ directory in the faas repo with this repo's root directory?

No, external providers need to run in their own microservice with the external_provider_url set on the gateway. See the faas-netes example to try this approach out.
